Library/Homebrew/test/formula_auditor_spec.rb

```diff
@@ -0,0 +1,1295 @@
+# frozen_string_literal: true
+
+require "formula_auditor"
+
+module Count
+  def self.increment
+    @count ||= 0
+    @count += 1
+  end
+end
```

Comment:
This should probably be most to somewhere in `test/support`.

Reply:
eh, this seems unnecessary altogether, i removed it in the latest commit